lib/src/union_set_controller.dart

```diff
@@ -23,20 +23,21 @@ import 'union_set.dart';
 /// ```
 class UnionSetController<E> {
   /// The [UnionSet] that provides a view of the union of sets in [this].
-  UnionSet<E> get set => _set;
-  late final UnionSet<E> _set;
+  final UnionSet<E> set;
 
   /// The sets whose union is exposed through [set].
-  final _sets = <Set<E>>{};
+  final _sets;
```

Comment:
Can we add a type?
```suggestion
  final Set<Set<E>> _sets;
```

Reply:
Doh,  yes. That's why I prefer always writing the types, they keep working when you remove the initializer!